### DISCUSSION TITLE
add users with access to specific servers(and usernames)

### DISCUSSION THREAD

**Discussion by christospelekis** [gravitational/teleport#15318] [Q&A] [answered]
Hello,
i have search in the documentation but it is not clear how to do the following:
Setup more users (of cource i know to d othis via web interface)
Give to this users access to specific servers without option to see the Activity of this servers and without access to download files

It is any guide for this? At least to give access to specific servers.

**Comment by webvictim** [comment, marked as answer]
Take a look at this answer: https://github.com/gravitational/teleport/discussions/15146#discussioncomment-3317139

It isn't currently possible to restrict SCP access (i.e. file upload/download) via RBAC, this is being worked on.
